### DISCUSSION TITLE
Complete TypeScript Conversion - Raid

### DISCUSSION THREAD

**Discussion by JacobMGEvans** [nextauthjs/next-auth#1509] [Help]
Greetings,
If you are interested we can "Raid" your repository, there would be one big PR from our Organization, for a large specific task/problem domain; in this case TS conversion.

We would need a maintainer as a point of contact, we prefer using Discord https://discord.gg/xkdkAJZsBR

Let me know if you are interested.
You can read more about what we are doing here: https://github.com/OpenSourceRaidGuild/tutorial-quest
This is also an example of how we work and the workflow: HospitalRun/hospitalrun-frontend#2516

cc: @nobrayner @mpeyper @Aprillion

**Comment by iaincollins** [comment]
Out of respect for the time and effort of maintainers, please use the templates provided when raising an issue.

### RE: TypeScript

[A good place to start is probably looking at existing issues to see why this has been rejected before.](https://github.com/nextauthjs/next-auth/pulls?q=is%3Aclosed+typescript+)

**A tl;dr on this is that we are not looking to port this project to another language.**
